test/Object/rlexe.xml

```diff
@@ -453,19 +453,31 @@
   <test>
     <default>
       <files>ObjectSpread_Simple.js</files>
-      <compile-flags>-args summary -endargs -NoNative -ES2018ObjectSpread</compile-flags>
+      <compile-flags>-args summary -endargs -NoNative -ES2018ObjectRestSpread</compile-flags>
     </default>
   </test>
   <test>
     <default>
       <files>ObjectSpread_JIT.js</files>
-      <compile-flags>-args summary -endargs -ES2018ObjectSpread -bgjit -maxinterpretcount:1 -off:simplejit</compile-flags>
+      <compile-flags>-args summary -endargs -ES2018ObjectRestSpread -bgjit -maxinterpretcount:1 -off:simplejit</compile-flags>
     </default>
   </test>
   <test>
     <default>
       <files>ObjectSpread_Limits.js</files>
-      <compile-flags>-args summary -endargs -ES2018ObjectSpread</compile-flags>
+      <compile-flags>-args summary -endargs -ES2018ObjectRestSpread</compile-flags>
+    </default>
+  </test>
+  <test>
+    <default>
+      <files>ObjectRest_Simple.js</files>
+      <compile-flags>-args summary -endargs -NoNative -ES2018ObjectRestSpread</compile-flags>
+    </default>
+  </test>
+  <test>
+    <default>
+      <files>ObjectRest_JIT.js</files>
+      <compile-flags>-args summary -endargs -ES2018ObjectRestSpread -bgjit -maxinterpretcount:1 -off:simplejit</compile-flags>
```

Comment:
I think `-bgjit-` (with a trailing `-`) is probably preferable to ensure JIT happens on main thread (though I think `maxinterpretcount` might accomplish that goal anyway)